scripts/jenkins/set-node-version.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+
+# Install nvm for this shell
+source ~/.nvm/nvm.sh
+
+# Use node.js at 4.2.x
+nvm install 4.2
```

Comment:
From the node.js docs:

> **Minors:** Minors are the addition and refinements of APIs or subsystems. They do not generally change APIs or introduce backwards compatible breaking changes, except where unavoidable. These are mostly completely additive releases.

I think it is safer to stick with a fixed minor release for automatic updates